recipes/pennylane-lightning-gpu/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pennylane-lightning-gpu" %}
+{% set version = "0.29.0" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PennyLaneAI/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e4cb8c496bddbb6c6cd27a6e1462cd3625952996222185dd0d7ec82e3cafea9b
+  patches:
+    - 0001-setup.patch
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version !=  "11.2")]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - ninja
+  host:
+    - {{ pin_compatible('custatevec', max_pin='x') }}
+    - libgomp      # [linux]
```

Comment:
You have included libgomp, but the build logs show that "ENABLE_OPENMP" is OFF. CMake may be locating openmp support but not compiling your package with openmp enabled?

Reply:
> Can you explain more? You already have pip listed in test/requires, why does this package also depend on pip at runtime?

You are correct. This is not a requirement of this package. It is a requirement of pennylane, since there is a function that uses pip to produce a report on which plugins are installed. I'll remove it.

> You have included libgomp, but the build logs show that "ENABLE_OPENMP" is OFF. CMake may be locating openmp support but not compiling your package with openmp enabled?

`CMakeLists` uses `ENABLE_OPENMP` and `PLLGPU_ENABLE_OPENMP` to target Lightning and Lightning-GPU respectively. They are `OFF` and `ON` by default. If that's ok, I would just keep using the defaults.